build.gradle

```diff
@@ -14,7 +14,7 @@ project.ext {
     version_number = '1.0.0'
     openSamlVersion = '3.3.0'
     verifyCommonUtils = '2.0.0-326'
-    samlLibsVersion = "$openSamlVersion-104"
+    samlLibVersion = "$openSamlVersion-107"
```

Comment:
Open SAML libs have moved on now... 126

Reply:
maybe, but that's out of scope for this PR

Reply:
we can raise another PR about that later